index.html

```diff
@@ -0,0 +1,180 @@
+    <!-- home page ends here -->
+    <!-- cards starts here -->
+    <section id="cards" class="cards">
+      <p>Main Program</p>
+      <hr class="about-hr" />
+      <div class="card-group">
+        <div class="card">
+          <img src="./assests/lecture.png" class="card-img-top" alt="card" />
+          <div class="card-body">
+            <h5 class="card-title">Lecture</h5>
+            <p class="card-text">
+              Listen to the speakers from the various countries about the
+              messages of sharing and opening
+            </p>
+          </div>
+        </div>
+        <div class="card">
+          <img src="./assests/Exhibition.png" class="card-img-top" alt="card" />
+          <div class="card-body">
+            <h5 class="card-title">CC Exhibition</h5>
+            <p class="card-text">
+              Appreacite various creations applying CC licence of artists,
+              organized from DMG events.
+            </p>
+          </div>
+        </div>
+        <div class="card">
+          <img src="./assests/forum.png" class="card-img-top" alt="card" />
+          <div class="card-body">
+            <h5 class="card-title">Forum</h5>
+            <p class="card-text">
+              Have the time to share your thoughts and opnions with experts for
+              each topic.
+            </p>
+          </div>
+        </div>
+        <div class="card">
+          <img src="./assests/workshop.png" class="card-img-top" alt="card" />
+          <div class="card-body">
+            <h5 class="card-title">Workshop</h5>
+            <p class="card-text">
+              Try creating your own work using open source license rather than
+              just watching it.
+            </p>
+          </div>
+        </div>
+        <div class="card">
+          <img src="./assests/ignite.png" class="card-img-top" alt="card" />
+          <div class="card-body">
+            <h5 class="card-title">CC Ignite</h5>
+            <p class="card-text">
+              Get opportunities to netwrok with CC affiliates around the world,
+              also after the summit.
+            </p>
+          </div>
+        </div>
+      </div>
+      <button type="button">Join Urban Design and Landscape 2023</button>
+    </section>
+    <!-- cards end here  -->
+    <!-- partners section starts here  -->
+    <section class="partner" id="partner">
+      <h1>Partners</h1>
+      <hr class="about-hr" />
+      <div class="partner-images">
+        <img src="./assests/mozilla.png" alt="mozilla" />
+        <img src="./assests/google.png" alt="google" />
+        <img src="./assests/naver.png" alt="naver" />
+        <img src="./assests/daumkakao.png" alt="daumkakao" />
+        <img src="./assests/airbnb.png" alt="airbnb" />
+      </div>
+    </section>
+    <!-- partners section ends here  -->
+
+    <!-- footer starts here  -->
+    <footer>
+      <div class="cc">
+        <img src="./assests/cc.png" alt="creative-common" />
```

Comment:
- [ ] please, the footer of the desktop home page has a white background. Kindly fix that
yours
![Screenshot (1197)](https://github.com/Kaiserabbas/Capstone/assets/65298048/84249d5f-6f8d-46f4-ab68-e3dc83996870)
original
![Screenshot (1196)](https://github.com/Kaiserabbas/Capstone/assets/65298048/ab1688ee-a7cd-4bf2-93b7-a75167b44643)